aries-site/src/layouts/main/Layout.js

```diff
@@ -77,24 +83,38 @@ export const Layout = ({
             <MainContentWrapper>
               <Main overflow="visible">
                 <SkipLinkTarget id="main" />
-                <Box>
+                {/* row-reverse direction to tabs through ToC first */}
+                <Box direction={layout !== 'plain' ? 'row-reverse' : 'column'}>
                   {layout !== 'plain' ? (
                     <>
-                      <ContentSection>
-                        <DocsPageHeader
-                          title={title}
-                          topic={topic}
-                          render={render}
-                        />
-                        {children}
-                      </ContentSection>
-                      {relatedContent.length > 0 && (
-                        <RelatedContent
-                          relatedContent={relatedContent}
-                          title={title}
-                        />
-                      )}
-                      <FeedbackSection />
+                      {!['xsmall', 'small'].includes(size) ? (
+                        <Box pad={{ left: 'large' }}>
+                          <InPageNavigation title={title} />
+                        </Box>
+                      ) : undefined}
+                      <Box
+                        width={
+                          !['xsmall', 'small'].includes(size)
+                            ? 'calc(100% - 192px)'
```

Comment:
Can we add a small comment here that `192px` is equivalent to `small` t-shirt size?